### PR TITLE
Fix bug spawning with the same key

### PR DESCRIPTION
**Pull request by paulgb**

### CODE REVIEW

File: plane/plane-tests/Cargo.toml [hunk 1]
```diff
@@ -4,19 +4,19 @@ version = "0.2.0"
...
 reqwest = { version = "0.11.22", features = ["json", "rustls-tls"], default-features = false }
+serde_json = "1.0.107"
 thiserror = "1.0.50"
-hyper = { version = "0.14.27", features = ["server"] }
-anyhow = "1.0.75"
+tokio = { version = "1.33.0", features = ["macros", "rt-multi-thread", "signal"] }
+tracing = "0.1.40"
+tracing-appender = "0.2.2"
+tracing-subscriber = { version = "0.3.17", features = ["env-filter"] }
 url = "2.4.1"
```
Comment (paulgb) [inline plane/plane-tests/Cargo.toml:22]: this is an unrelated change, I just sorted the deps

**Review by rolyatmax — APPROVED** [review]
❤️
